### ISSUE TITLE
traceback: created_date = photo.created.astimezone(get_localzone()) OSError: [Errno 22] Invalid argument

### ISSUE THREAD

**Issue by ekinsol** [ndbroadbent/icloud_photos_downloader#120]
Hi,

I get this error around 72% (on every run):

HEVC.MOV already exists.:  21%|##9           | 4498/20995 [01:49<06:59, 39.28it/s] E:\icloud_photos\2015/07/16\IMG_4901.JPG already exists.:  72%|############9     | 15074/20995 [05:27<01:32, 63.88it/s]Traceback (most recent call last):
  File "c:\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python37\Scripts\icloudpd.exe\__main__.py", line 9, in <module>
  File "c:\python37\lib\site-packages\click\core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "c:\python37\lib\site-packages\click\core.py", line 697, in main
    rv = self.invoke(ctx)
  File "c:\python37\lib\site-packages\click\core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\python37\lib\site-packages\click\core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "c:\python37\lib\site-packages\icloudpd\base.py", line 335, in main
    created_date = photo.created.astimezone(get_localzone())
OSError: [Errno 22] Invalid argument

It seems to be related to the created_date

Kind Regards,
Nicky

PS: Already glad I managed to fetch all the other files ;-)

**Comment by ndbroadbent** [comment]
Hello, sorry about that! I was already catching a `ValueError` here for invalid dates, but looks like Python can [also throw an `OSError`](https://stackoverflow.com/questions/37494983/python-fromtimestamp-oserror) sometimes.

I've released version `1.4.2` which should fix this issue. Instead of a crash, you'll see an error message like this:

```
Could not convert photo created date to local timezone (...)
```

Hope that helps!

**Comment by ekinsol** [comment]
> Hello, sorry about that! I was already catching a `ValueError` here for invalid dates, but looks like Python can [also throw an `OSError`](https://stackoverflow.com/questions/37494983/python-fromtimestamp-oserror) sometimes.
> 
> I've released version `1.4.2` which should fix this issue. Instead of a crash, you'll see an error message like this:
> 
> ```
> Could not convert photo created date to local timezone (...)
> ```
> Hope that helps!

It sure helps, now on 73%! Thank you very much :-)

Merry Christmas ;-)
